src/portal/angular.json

```diff
@@ -40,9 +40,10 @@
               }
```

Comment:
![image](https://user-images.githubusercontent.com/29826789/94092120-7ad57300-fe4c-11ea-8b90-00a03d50dd5a.png)
Is this alignment problem caused by the upgrade?

Reply:
![image](https://user-images.githubusercontent.com/29826789/94092173-a0627c80-fe4c-11ea-98a6-df6f70a69a46.png)

Reply:
Fixed